tests/e2e/features/3DViewer.feature

```diff
@@ -18,17 +19,17 @@ Feature: preview 3D model
         # toggle between standard and fullscreen mode
         When the user enters fullscreen mode
         Then the 3D model should be displayed in fullscreen mode
-        And the topbar should not be visible
+        # And the topbar should not be visible
```

Comment:
@saw-jan 
this step can't be tested with the current implementation because the HTML element is actually there, it's just behind the enlarged canvas... can I delete it from the scenario?

Reply:
yeah, step `Then the 3D model should be displayed in fullscreen mode` should be enough